dataverse-webapp/src/main/webapp/dataset.xhtml

```diff
@@ -601,10 +601,9 @@
                                          update=":datasetForm:topDatasetBlockFragment,:datasetForm:tabView:filesTable,:messagePanel"
                                          actionListener="#{DatasetPage.refresh}"/>
 
-                        <!-- see the comment in the javascript below that explains what this button is for -->
-                        <p:commandButton id="refreshButton" process="@this" widgetVar="refreshButton"
-                                         update=":datasetForm:topDatasetBlockFragment,:datasetForm:tabView:filesTable,:messagePanel,datasetForm:tabView:editMetadataButton"
-                                         style="display:none"/>
+                        <p:remoteCommand name="pageUnlockedInfo" process="@this"
+                                         update=":messagePanel"
```

Comment:
Our messages.xhtml contains 'autoupdate' so I think it is not needed to update it.